packages/gatsby-source-wordpress/src/steps/declare-plugin-options-schema.js

```diff
@@ -84,14 +84,11 @@ const pluginOptionsSchema = ({ Joi }) => {
               }
             `),
         }),
-      beforeChangeNode: Joi.any()
+      beforeChangeNode: Joi.string()
         .allow(null)
         .allow(false)
-        .meta({
-          trueType: `function`,
-        })
         .description(
-          `A function which is invoked before a node is created, updated, or deleted. This is a hook in point to modify the node or perform side-effects related to it.`
+          `A function which is invoked before a node is created, updated, or deleted. This is a hook in point to modify the node or perform side-effects related to it. This option should be a path to a JS file where the default export is the beforeChangeNode function. The path can be relative to your gatsby-node.js or absolute. Currently you can inline a function by writing it out directly in this option but starting from Gatsby v4 only a path to a function file will work.`
```

Comment:
```suggestion
          `A function which is invoked before a node is created, updated, or deleted. This is a hook in point to modify the node or perform side-effects related to it. This option should be a path to a JS file where the default export is the beforeChangeNode function. The path can be relative to your gatsby-node.js or absolute. Currently you can inline a function by writing it out directly in this option but starting from Gatsby v4 only a path to a function file will work.`
```

same as the other comment

Reply:
fwiw the plugin-options.md file is generated from this file :)